src/api/route-services/samples.js

```diff
@@ -144,6 +145,65 @@ class SamplesService {
     return OK();
   }
 
+  async addSample(projectUuid, experimentId, newSample) {
+    logger.log(`Adding samples ${JSON.stringify(newSample)} to project with id ${projectUuid}`);
```

Comment:
Wouldn't doing `JSON.stringify` cause this log to be long? Is it not enough to just print the `sampleUuid`?